contrib/kafka-container-source/samples/sources_v1alpha2_kafka_container_source.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: sources.eventing.knative.dev/v1alpha1
+kind: ContainerSource
+metadata:
+  name: my-kafka-source
+spec:
+  image: github.com/knative/eventing-sources/contrib/kafka-container-source/cmd
+  args: 
+    - '--bootstrap=kafkabroker.kafka:9092'
```

Comment:
Curious about the `args` here. Seems the other event sources have custom defined labels for arguments instead of using the `args` field.